src/components/ConnectedTile.jsx

```diff
@@ -38,10 +36,22 @@ const stateIcon = status => {
 }
 
 const mapStateToProps = (state, props) => {
-  return {
-    icon: stateIcon(
-      getConnectionStatusForTrigger(state.connections, props.trigger)
+  if (props.trigger) {
+    return {
+      icon: stateIcon(
+        getConnectionStatusForTrigger(state.connections, props.trigger)
+      )
+    }
+  } else {
+    // no trigger provided, we don't display a connection
```

Comment:
That's the part I was afraid of. I am totally not comfortable with having a `if/else` returning totally different kind of props right here. In my mind we should keep minimal component with almost no logic in it, so I was more thinking about a new kind of tile, for example `ConnectionsTile` which encapsulate a `KonnectorTile`. This solution needs to improve `KonnectorTile` to be able to display an icon or something else like maybe a component for the counter ? What do you think ? I would love to have @goldoraf opinion about this.